driver-reactive-streams/build.gradle

```diff
@@ -75,6 +75,7 @@ afterEvaluate {
     jar.manifest.attributes['Bundle-SymbolicName'] = 'org.mongodb.driver-reactivestreams'
     jar.manifest.attributes['Import-Package'] = [
             'com.mongodb.crypt.capi.*;resolution:=optional',
+            'com.mongodb.internal.crypt.capi.*;resolution:=optional',
```

Comment:
reactive-streams (and sync) reference `MongoCryptException`, and also the classes that were moved into the internal package, so both packages are now needed.